pandas/core/groupby/groupby.py

```diff
@@ -1916,11 +1952,12 @@ def ngroup(self, ascending=True):
             index = self._selected_obj.index
             result = Series(self.grouper.group_info[0], index)
             if not ascending:
-                result = self.ngroups - 1 - result
+                result = self.ngroups - result
```

Comment:
This change is breaking `pandas/tests/groupby/test_counting.py::TestCounting`

Reply:
Yea thanks that was a mistake changing in the first place; this generates a mypy I was trying to debug but haven't seen all of the way through yet